miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -788,76 +819,38 @@ protected LibraryDto packageDtoFromParent(Sample sample) {
     }
 
     @Override
-    protected Iterable<Library> load(List<Long> ids) throws IOException {
-      List<Library> results = libraryService.listByIdList(ids);
-      SampleClass sampleClass = null;
-      for (Library library : results) {
-        if (!isDetailedSampleEnabled()) {
-          // Do nothing about sample classes.
-        } else if (sampleClass == null) {
-          sampleClass = ((DetailedSample) library.getSample()).getSampleClass();
-        } else if (((DetailedSample) library.getSample()).getSampleClass().getId() != sampleClass.getId()) {
-          throw new IOException("Can only update libraries when samples all have the same class.");
-        }
-      }
-      return results;
-    }
-
-    @Override
-    protected LibraryDto asDto(Library model) {
-      return Dtos.asDto(model);
-    }
-
-    @Override
-    protected Class<? extends LibraryDto> getDtoClass() {
-      return isDetailedSampleEnabled() ? DetailedLibraryDto.class : LibraryDto.class;
-    }
-
-    @Override
     protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) {
-      config.put("showDescription", showDescription);
-      config.put("showVolume", showVolume);
-      config.put("showLibraryAlias", showLibraryAlias);
+      writeLibraryConfiguration(mapper, config);
```

Comment:
Oh, sorry. Missed a curly brace and thought `writeLibraryConfiguration` was also inside of `libraryBulkPropagateBackend`. :+1: